### DISCUSSION TITLE
AdminUsersFeature UI Click the edit button in the user list. The first click is invalid.

### DISCUSSION THREAD

**Discussion by LEIRONGHUA** [ServiceStack/Discuss#89] [Q&A] [answered]
When modifying a user, click the edit button in the table below. The first click is invalid. It takes a total of two clicks to switch the modification record.

**Environment：**

MySQL Version：8.0.31
ServiceStack.Server Version：6.11.1
AspNetCore Version：7.0

**Output:**

![动画](https://github.com/ServiceStack/Discuss/assets/4002277/9b9ddfc0-e84f-4cb1-bfd0-b008f976d056)

Please check this issue, Thanks

**Comment by mythz** [comment, marked as answer]
Should be resolved from the latest v6.11.1+ that's now available in [Pre release packages](https://docs.servicestack.net/pre-release). 

If you already have v6.11.1 installed you'll need to [Clear your NuGet packages cache](https://docs.servicestack.net/pre-release#clear-nuget-package-cache) to download the latest version.
